pil-analyzer/tests/types.rs

```diff
@@ -480,7 +480,7 @@ fn enum_pattern() {
 }
 
 #[test]
-#[should_panic = "Only one \"..\"-item allowed in array pattern"]
+#[should_panic = "Only one \\\"..\\\"-item allowed in array pattern"]
```

Comment:
Is this change correct?

Reply:
Is the way it's parsed now that the error is a proper error and not an string, any ideas on how to solve it?